src/PKSim.Core/Model/ExpressionProfileFactory.cs

```diff
@@ -50,6 +55,7 @@ public ExpressionProfile Create(Type moleculeType, Species species, string molec
       {
          var expressionProfile = _objectBaseFactory.Create<ExpressionProfile>();
          expressionProfile.IsLoaded = true;
+         expressionProfile.Category = _usedExpressionProfileCategoryRepository.All().FirstOrDefault();
```

Comment:
Default is not always healthy but the first one in the list. I think it makes much more sense if the project does not deal with healthy individual and one always has to change the value to sthg that was input previously. So either empty,. or this